api/app.ts

```diff
@@ -29,6 +29,8 @@ app.post('/changePassword', changePasswordEndpoint);
 
 import accountCreate from './accounts/create';
 app.post('/accounts/create', accountCreate);
+import resetPassword from './accounts/resetPassword';
+app.post('/accounts/resetPassword', resetPassword);
```

Comment:
I'm all for organizing these like this, but right now it seems more unorganized to have two endpoints randomly at /accounts/
Also the users CRUD service just posts to /resetPassword.

Reply:
Semantically, that endpoint is correct. The CRUD service should point to /accounts/resetPassword. (Because we are resetting a specific account's password.)

When I connect the _mockup => service => endpoint_ chain, I'll change the endpoint as needed.